### DISCUSSION TITLE
Samsung Tab A Android 7

### DISCUSSION THREAD

**Discussion by FBukov** [cagnulein/qdomyos-zwift#1666] [Q&A] [answered]
Hi,
I hooked up my Echelon Sport to the QZwift app. It finds the bike, no problems, also the external heart monitor. It also shows the resistance level. But when I start pedalling, it does not show power output, cadence nor the distance.
Any suggestions? I enabled Ant in the Ant Settings, because I want to use it as a sensor on my Garmin Fenix.
Thanks for the replies.
Regards

**Comment by cagnulein** [comment]
Send me a debug log and i will check it
https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

Il giorno dom 24 set 2023 alle 23:55 FBukov ***@***.***> ha
scritto:

**Reply by FBukov** [reply]
Here are the files. Started an activity, heart and resistance are being read, but that's it.
![Mon  Sep  25 13_29_43 2023](https://github.com/cagnulein/qdomyos-zwift/assets/145798557/146620aa-f997-45ab-8f30-8f2f8bfea5ae)
[debug-Mon_Sep_25_06_47_53_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/12718517/debug-Mon_Sep_25_06_47_53_2023.log)
[debug-Mon_Sep_25_13_24_02_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/12718518/debug-Mon_Sep_25_13_24_02_2023.log)
[debug-Mon_Sep_25_13_27_29_2023.zip](https://github.com/cagnulein/qdomyos-zwift/files/12718519/debug-Mon_Sep_25_13_27_29_2023.zip)

**Reply by cagnulein** [reply, marked as answer]
@FBukov cadence sensor and power sensor must be set to disabled, that's the issue 

let me know!

**Reply by FBukov** [reply]
in ANT settings?

**Comment by cagnulein** [comment]
no under accessories

Il giorno lun 25 set 2023 alle 21:25 FBukov ***@***.***> ha
scritto:

**Reply by FBukov** [reply]
it did work. one more question; for the speed sensor, what value should I set for the wheel circumference on Echelon Sport?

**Reply by cagnulein** [reply]
100mm on the watch

**Reply by FBukov** [reply]
thanks, will let you know if everything works with the watch now that I have data on android
